pages/index.tsx

```diff
@@ -0,0 +1,41 @@
+import type { NextPage } from "next";
+import Image from "next/image";
+import FeatureGrid from "../components/branding/FeatureGrid";
+import OpenGraph from "../components/branding/util/OpenGraph";
+
+const Home: NextPage = () => {
+  return (
+    <>
+      <OpenGraph
+        title="Imageing"
+        url="https://imageing.org/"
+        image="https://imageing.org/assets/Mark.png"
+        description="request changes here too!"
+      />
+
+      <div className="hero">
+        <div className="hero-content text-center">
+          <div className="max-w-md">
+            <Image
+              className="hover:scale-110 transition-all duration-500 ease-in-out hover:drop-shadow-2xl"
+              src="/assets/Mark.png"
+              alt="Splash"
+              width={200}
+              height={200}
+            />
+            <p className="py-6">
+              some app to do with images, i wasn't given info. (request changes
```

Comment:
Read the fucking repo readme

Reply:
https://github.com/ImageingApp/Website/commit/edb72a06e48b0d94b9ed95bebb0aee6c1bf654c6